2024 results (#134)
geeksblabla/stateofdev.ma: results/2024/sections/community.mdx

---
title: Community
position: 7
---

# Community

Moroccan Developers continue to be increasingly involved in open source. A significant portion of respondents have contributed to open-source projects.

## Local Developer Communities

Many respondents follow local communities, though active participation remains a challenge.

<Chart id="community-q-0" title={false} />

## Open source contribution

Open source contribution continues to grow among Moroccan developers.

<Tabs>
  <TabItem title="2024">
    <Chart id="community-q-1" title={false} year={2024} />
  </TabItem>
  <TabItem title="2023">
    <Chart id="community-q-1" title={false} year={2023} />
  </TabItem>
  <TabItem title="2022">
    <Chart id="community-q-1" title={false} year={2022} />
  </TabItem>
</Tabs>

## Blogging

Many respondents plan to write blog posts in the future, though only a portion have had the opportunity to do so.

<Chart id="community-q-2" title={false} />

## Social Media

> LinkedIn remains the most used social media network among Moroccan developers to connect and get the latest Tech news, followed by Twitter/X and Discord.

Reviewer: ```suggestion
LinkedIn remains the most used social media network among Moroccan developers to connect and get the latest Tech news, followed by Twitter/X and Youtube.
```

<Chart id="community-q-3" title={false} />

## Tech events attended in 2024

> A significant portion of respondents attended at least one tech event in the past year.

Reviewer: ```suggestion
Slightly more than half the respondents haven't attended a tech event in the past year.
```
